modules/gcp-velero/iam.tf

```diff
@@ -4,12 +4,50 @@
  * license that can be found in the LICENSE file.
  */
 
+locals {
+  velero_role_permissions = [
+    "compute.disks.get",
+    "compute.disks.list",
+    "compute.disks.create",
+    "compute.disks.createSnapshot",
+    "compute.snapshots.get",
+    "compute.snapshots.list",
+    "compute.snapshots.create",
+    "compute.snapshots.useReadOnly",
+    "compute.snapshots.delete",
+    "compute.zones.get"
+  ]
+}
+
 resource "google_service_account" "velero" {
   project      = var.project
-  account_id   = "${var.backup_bucket_name}-velero"
-  display_name = "Velero account for ${var.backup_bucket_name}"
+  account_id   = var.gcp_service_account_name
+  display_name = "Service account for Velero"
 }
 
 resource "google_service_account_key" "velero" {
   service_account_id = google_service_account.velero.name
 }
+
+resource "google_project_iam_custom_role" "velero-role" {
```

Comment:
Consider using underscore resource names to be aligned with all the resources in the terraform project

Reply:
✅  thanks